packages/zowe-explorer/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `2.3.1`
```

Comment:
it'd be nice if we used `## TBD Release` or something like that.
That way we don't have to dictate which version we believe this PR will be published in.
For example, there may be other PRs already in (or in route to) `main` that introduce enhancements.